Tests/QuickTests/QuickTests/FunctionalTests/CurrentSpecTests.swift

```diff
@@ -2,12 +2,18 @@ import Quick
 import Nimble
 import Dispatch
 
-#if canImport(Darwin)
-
 class CurrentSpecTests: QuickSpec {
     override func spec() {
         it("returns the currently executing spec") {
-            expect(QuickSpec.current.name).to(match("currently_executing_spec"))
+            let name: String = {
+                let result = QuickSpec.current.name
+                #if canImport(Darwin)
+                return result.replacingOccurrences(of: "_", with: " ")
+                #else
+                return result
```

Comment:
Because we don't use example name to selector name conversion on Linux (there is no Objective-C runtime off-course 😃), we don't need to care about whether white spaces are used or not in example names.